### PR TITLE
Make client classes Isolated

### PR DESCRIPTION
**Pull request by lahirusamith**
## Purpose
$Title 

#### related PRs are:  https://github.com/ballerina-platform/module-ballerina-sql/pull/245

Fixes:

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -5,3 +5,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Changed
+- Make DB Client class isolated
```
Comment (niveathika) [inline changelog.md:10]:
```suggestion
- Make JDBC Client class isolated.
```

Comment (lahirusamith) [reply changelog.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/175/commits/c7bfa6b487a3c7a3f3dba6af9a91eaaa114bb2fa

**Review by niveathika — APPROVED** [review]
@lahirusamith Let's bump the SQL version once it's released and merge this.

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**